app/_data/version_errors_konnect.yml

```diff
@@ -1934,2278 +1934,4158 @@ messages:
+      consumer_group. This is not supported in Kong Gateway versions < 3.9.0.0. The
+      plugin has been removed in the data plane node.
     Resolution: |
-      To ensure continued Konnect support, please upgrade to the latest 2.8 LTS
-      version.
+      Unset the `consumer_group` fields for the plugin, or upgrade Kong Gateway to a
+      version >= 3.9.0.0.
     DocumentationURL: |
-      [Compatibility](/konnect-platform/compatibility/)
-  - ID: EOL101
-    Severity: error
+      [AI Semantic Cache plugin](/plugins/ai-semantic-cache/)
+  - ID: D284
+    Severity: warning
     Description: |
-      Kong Gateway version 2.6.x.x has now reached its End of Full Support, and after
-      July 31, 2024, data plane connections for this version will no longer receive
-      configurations from Konnect.
+      A route expression is using deprecated fields
     Resolution: |
-      To ensure continued Konnect support, please upgrade to the latest 2.8 LTS
-      version.
+      Replace deprecated features with currently preferred ones
     DocumentationURL: |
-      [Compatibility](/konnect-platform/compatibility/)
-  - ID: EOL102
+      [Expressions Router](/gateway/routing/expressions)
+  - ID: D285
     Severity: error
     Description: |
-      Kong Gateway version 2.7.x.x has now reached its End of Full Support, and after
-      July 31, 2024, data plane connections for this version will no longer receive
-      configurations from Konnect.
+      For any `ai-proxy-advanced`, `ai-semantic-cache`, or `ai-semantic-prompt-guard`
+      plugins, the `config.embeddings.model.name` field has been set to a free-form
+      text value. This is not supported in Kong Gateway versions < 3.9.0.0. The plugin
+      configuration has been updated to use the default embeddings model for the
+      selected provider.
     Resolution: |
-      To ensure continued Konnect support, please upgrade to the latest 2.8 LTS
-      version.
+      Please upgrade Kong Gateway to version `3.9.0.0` or above.
     DocumentationURL: |
-      [Compatibility](/konnect-platform/compatibility/)
-  - ID: EOL103
-    Severity: error
+      [AI Proxy Advanced plugin](/plugins/ai-proxy-advanced/)
+  - ID: D286
+    Severity: warning
     Description: |
-      Kong Gateway version 3.0.x.x has now reached its End of Full Support, and after
-      July 31, 2024, data plane connections for this version will no longer receive
-      configurations from Konnect.
+      One or more of the following plugins are not supported for Cloud Gateways:
+      `file-log`, `syslog`, `app-dynamics`, `prometheus`.
     Resolution: |
-      To ensure continued Konnect support, please upgrade to the latest 3.4 LTS
-      version.
+      Remove the unsupported plugins
     DocumentationURL: |
-      [Compatibility](/konnect-platform/compatibility/)
-  - ID: EOL104
+      [Unsupported Plugins in Cloud Gateways](/konnect/gateway-management/cloud-gateway/#unsupported-plugins)
```

Comment:
this url returns 404, where did you get it from?

Reply:
I think an alternative would be https://developer.konghq.com/plugins/compatibility/